Core/source/test.ts

```diff
@@ -1086,6 +1092,7 @@ ava.default('decryptErr for not integrity protected message', async t => {
       [await getCompatAsset('mime-email-not-integrity-protected')]));
   expect(decryptJson.replyType).equals('plain');
   expect(decryptJson.subject).equals('not integrity protected - should show a warning and not decrypt automatically');
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
this one also seems unnecessary

Reply:
The same error with `JSON.parse` here too

Reply:
Got it.
In this case, maybe we can change code like this?
```ts
const blocks = decryptData.toString().split('\n').map(block => JSON.parse(block) as MsgBlock);
  expect(blocks[1].decryptErr?.error.type).equals('no_mdc');
```

Reply:
My pleasure :)